tests/integration/test_powershell_tool/test_terminal_timeout.py

```
@@ -457,7 +457,11 @@ def test_process_communication_health(self):
                     print(f"❌ Command {i} failed: {e}")
             # Check process state after tests
             print(f"Process still alive: {session._process.poll() is None}")
+if session._process is not None:
```

Ruff (0.14.3)

Undefined name `session`

(F821)